test/e2e/network/hostport.go

```diff
@@ -146,6 +146,7 @@ var _ = SIGDescribe("HostPort", func() {
 				continue
 			}
 			success = true
+			break
```

Comment:
I don't know what I was thinking when I did this :smile: , thanks for fixing it

an alternative can be to not use the `success` variable and return directly here, and fail if we go out of the loop
up to you